sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -495,17 +499,19 @@ public void close() {
             messages.add(message);
         }
 
-        final Context finalSharedContext = sharedContext != null ? sharedContext : Context.NONE;
+        if (isTracingEnabled) {
+            final Context finalSharedContext = sharedContext == null
+                ? Context.NONE
+                : sharedContext
+                    .addData(ENTITY_PATH_KEY, "entityPath")
+                    .addData(HOST_NAME_KEY, "hostName")
```

Comment:
For the entityPath and hostName values, do we want to update the `ServiceBusSenderAsyncClient` or we should be able to retrieve it some other way?

Reply:
There is an `entityName` you can use for `entityPath`. I'm sure fullyqualifedDomainName is also exposed in this class.